src/common/platform/posix/sdl/i_system.cpp

```diff
@@ -175,6 +175,7 @@ void I_PrintStr(const char *cp)
 {
 	const char * srcp = cp;
 	FString printData = "";
+	bool terminal = isatty(stdout);
```

Comment:
`isatty()` function operates on file descriptors, not pointers to `FILE`.

Reply:
Fixed.